crates/worker/src/partition_processor_manager.rs

```diff
@@ -607,6 +642,23 @@ impl PartitionProcessorManager {
         let bifrost = self.bifrost.clone();
         let node_id = self.metadata.my_node_id();
 
+        let schema = self.metadata.updateable_schema();
+
+        let invoker: InvokerService<
+            InvokerStorageReader<PartitionStore>,
+            EntryEnricher<Schema, ProtobufRawEntryCodec>,
+            Schema,
+        > = InvokerService::from_options(
+            &config.common.service_client,
+            &config.worker.invoker,
+            EntryEnricher::new(schema.clone()),
+            schema.clone(),
+        )
+        .unwrap();
```

Comment:
The original code handles failures gracefully if we cannot create the invoker service. I added the unwrap as a quick hack. Should we bubble up errors instead of panicking?

Reply:
yeah, definitely.